lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
I'd prefer to generate a new file for each channel subscription so we're going to need at least one global for the consumer instance. If we use `Cable` just for that (`Cable.consumer = Cable.createConsumer(...)`) then we don't have to think about the user's global `AppName` object at all and free ourselves from potential conflicts, naming assumptions, and imposing more globals in their app.

So, we'd generate:
```coffee
# app/assets/javascripts/cable/index.coffee

#= require cable
#= require_self
#= require_tree .

Cable.consumer = Cable.createConsumer(...)
```
and
```coffee
# app/assets/javascripts/cable/subscriptions/appearance.coffee
appearance = Cable.consumer.subscriptions.create "AppearanceChannel",
  ...
```